CHANGELOG.md

```diff
@@ -32,6 +32,10 @@ The `IX_WORKFLOW_STORE_ENTRY_WS` index is removed from `WORKFLOW_STORE_ENTRY`.
 
 The index had low cardinality and workflow pickup is faster without it. Migration time depends on workflow store size, but should be very fast for most installations. Terminal workflows are removed from the workflow store, so only running workflows contribute to the cost.
 
+### Bug Fixes
+
+* Fixed `google_project` and `google_compute_service_account` workflow options not taking effect when using GCP Batch backend
```

Comment:
There were a few fixes included in the `GCP Batch` section, should we move this?

Reply:
Good point. Will fix!